kep/108-Memory-bandwidth-and-LLC-aware-filtering-and-scoring/README.md

```diff
@@ -0,0 +1,145 @@
+# Memory bandwidth and LLC aware filtering and scoring
```

Comment:
Please add a `kep.yaml` to include the KEP metadata.

Reply:
done